ui/apps/platform/src/Containers/Dashboard/PatternFly/Widgets/ComplianceLevelsByStandard.test.tsx

```diff
@@ -13,6 +13,34 @@ import ComplianceLevelsByStandard from './ComplianceLevelsByStandard';
 
 jest.setTimeout(10000);
 
+const standards = [
+    /*
+    These standards have been formatted for easier verification of the expected ordering in
+    tests compared to a direct hard coding in the mocked response below.
+
+    [numFailing, numPassing, id]
```

Comment:
What do you think about an order of items so reader even more quickly sees that `standards` `standardNames` have parallel structure?
* Replace `[numFailing, numPassing, id]` with `[id, numFailing, numPassing]` here.
* Replace `[, , id]` with `[id]` below.

Reply:
Optional extra credit: declare that strings in `standards` are keys in `standardNames` object?

Reply:
The `standards` const existed before the `standardNames`, with the goal of listing them by `numFailures` to easily tell what the intended ordering would be. (Although having them intentionally out of order would be a better test of the client side sorting...)

To your point though, it might be better to just combine the two now into something like:
```typescript
// id: [name, numFailing, numPassing]
const standards = {
    CIS_Docker_v1_2_0: ['CIS Docker v1.2.0', 9, 1],
    ...
};
```
and iterate with `Object.entries` when needed.